src/pages/Login.vue

```diff
@@ -1,87 +1,112 @@
-            />
-
-            <q-btn
-              rounded
-              :loading="loading"
-              :label="$t('login')"
-              v-on:click="login()"
-            ></q-btn>
-            <q-btn rounded :label="$t('register')" to="/register"></q-btn>
-          </form>
+    <q-card class="bg-black flex flex-center">
+      <q-img src="~assets/CoronaHelfer-Logo.svg"></q-img>
+    </q-card>
+
+    <q-card>
+      <form>
+        <div v-if="error !== ''" class="error">{{ error }}</div>
+        <q-input
+          filled
+          bg-color="yellow-1"
+          :label="$t('emailOrPassword')"
+          v-model="name"
+        />
+        <q-input
+          filled
+          bg-color="yellow-1"
+          type="password"
+          :label="$t('password')"
+          v-model="password"
+        />
+
+        <div class="row">
+          <q-btn
+            color="primary"
+            :loading="loading"
+            :label="$t('login')"
+            v-on:click="login()"
+          ></q-btn>
+          <q-btn
+            flat
+            :label="$t('orRegister')"
+            to="/register"></q-btn>
         </div>
-      </div>
-    </body>
+
+        <button class="oauth facebook row">
+          <div><q-icon name="fab fa-facebook-f"></q-icon></div>
+          <div>{{$t('loginFacebook')}}</div>
+        </button>
+
+        <button class="oauth google row">
+          <div><q-icon name="fab fa-google"></q-icon></div>
+          <div>{{$t('loginGoogle')}}</div>
+        </button>
+      </form>
+    </q-card>
+    <img class="logo" width="100"  />
   </q-page>
 </template>
 
 <style lang="sass" scoped>
-body
-  background: url('../statics/images/background.jpg') no-repeat
-  background-size: cover
-
-.authenticate
-  padding: 20px
-  display: flex
+.q-page
   justify-content: center
   align-items: center
-  height: calc(100vh - 170px)
-  box-sizing: border-box
-  .container
-    width: 400px
-    .login, .register
-      background-color: WHITE
-      padding: 15px 30px
-      display: flex
-      flex-direction: column
-      border-radius: 25px
-      input
-        width: 100%
-        background-color: LIGHTGRAY
-        border: none
-        box-shadow: none
-        padding: 0 25px
-        height: 30px
-        box-sizing: border-box
-        margin-bottom: 5px
-        border-radius: 19px
-        &:focus
-          outline: none
-        &.spacer
-          margin-bottom: 15px
-      button, a
-        height: 40px
-        padding: 0
-        font-size: 15px
-        font-weight: 600
-        margin-top: 15px
-    .register
-      .aligner
-        display: flex
-        margin-bottom: 10px
-        .left
-          width: 40%
-          .avatar
-            background: url('../statics/images/avatar.jpg') no-repeat
-            background-size: cover
-            width: 65px
-            height: 65px
-            border-radius: 100%
+  display: flex
+
+.q-card:first-of-type
+  width: 34rem
+  right: 20rem
```

Comment:
I was too fast.
You should not used `rem` for `width`, `height`, `left`, or `right`